events/wc_dark_cult_events.txt

```diff
@@ -1086,9 +1145,336 @@ narrative_event = {
+
+		set_character_flag = hunted_by_garona
+		hidden_effect = { set_variable = { which = garona_kills value = 1 } }
+	}
+	option = {
+		name = EVTOPTB_WCDAC_5041
+		trigger = { has_character_flag = hunted_by_garona }
+	}
+	after = {
+		hidden_effect = { change_variable = { which = garona_kills value = 1 } }
+		if = {
+			limit = { check_variable = { which = garona_kills value = 5 } }
+			character_event = { id = WCDAC.5042 months = 6 }
+		}
+		else = {
+			character_event = { id = WCDAC.5041 months = 10 random = 90 }
 		}
 	}
 }
+character_event = {
+	id = WCDAC.5042
+	desc = EVTDESC_WCDAC_5042
+	picture = GFX_evt_ahnqiraj
+	border = GFX_event_normal_frame_intrigue
+
+	is_triggered_only = yes
+	
+	trigger = {
+		c_10620 = {
+			is_alive = yes
+			is_dying = no
+	
+			is_untouchable_trigger = no
+	
+			is_adult = yes
+			is_incapable = no
+			prisoner = no
+			is_inaccessible_trigger = no
+			
+			is_landed = no
+			
+			NOT = { religion = old_gods_worship }
+			NOT = { is_married = 52000 }
+			NOT = { obedient = 52000 }
+		}
+		character = 52000
+	}
+	
+	fail_trigger_effect = {
+		# Skip event
+		character_event = { id = WCDAC.5045 months = 12 }
+		clr_character_flag = hunted_by_garona
+	}
+
+	option = {
+		name = EVTOPTA_WCDAC_5042
+		
+		add_trait = travelling
+		diplomatic_immunity = yes
+		character_event = { id = WCDAC.5043 months = 14 }
+	}
+}
+character_event = {
+	id = WCDAC.5043
+	desc = EVTDESC_WCDAC_5043
+	picture = GFX_evt_ahnqiraj
+	border = GFX_event_normal_frame_intrigue
+
+	is_triggered_only = yes
+	
+	trigger = {
+		c_10620 = {
+			is_alive = yes
+			is_dying = no
+	
+			is_untouchable_trigger = no
+	
+			is_adult = yes
+			is_incapable = no
+			prisoner = no
+			is_inaccessible_trigger = no
+			
+			is_landed = no
+			
+			NOT = { religion = old_gods_worship }
+			NOT = { is_married = 52000 }
+			NOT = { obedient = 52000 }
+		}
+		character = 52000
+	}
+	
+	fail_trigger_effect = {
+		# Skip event
+		character_event = { id = WCDAC.5045 months = 12 }
+		clr_character_flag = hunted_by_garona
+		remove_trait = travelling
+		diplomatic_immunity = no
+	}
+
+	option = {
+		name = EVTOPTA_WCDAC_5043
+		
+		clr_character_flag = hunted_by_garona
+		remove_trait = travelling
+		diplomatic_immunity = no
+		event_target:target_garona = {
+			remove_trait = travelling
+			#hidden_effect = { remove_trait = unable_to_interact_trait }
+			diplomatic_immunity = no
+			move_character = PREV
+		}
+		reverse_imprison = event_target:target_garona
+		character_event = { id = WCDAC.5044 days = 1 }
+	}
+}
+character_event = {
+	id = WCDAC.5044
+	desc = EVTDESC_WCDAC_5044
+	picture = GFX_evt_ahnqiraj
+	border = GFX_event_normal_frame_intrigue
+
+	is_triggered_only = yes
+
+	option = {
+		name = EVTOPTA_WCDAC_5044
+		
+		reverse_opinion = { modifier = opinion_loyal_servant who = event_target:target_garona }
+		event_target:target_garona = { imprison = no }
+		character_event = { id = WCDAC.5045 days = 7 }
+	}
+}
+character_event = {
+	id = WCDAC.5045
```

Comment:
Emm? I don't follow the Old Gods anymore.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/86472627-7ff4bb80-bd50-11ea-9785-450aa378967a.png)

</details>